tools/distpartitioning/data_shuffle.py

```diff
@@ -92,12 +92,19 @@ def gen_node_data(rank, world_size, node_part_ids, ntid_ntype_map, nid_schema_ma
     gnid_start = 0
     gnid_end = 0
     for ntypeid in range(len(ntid_ntype_map)):
-        ntype_name = ntid_ntype_map[str(ntypeid)]
-        ntype_info = nid_schema_map[ntype_name]
-        type_start = int(ntype_info["data"][0][1])
-        type_end = int(ntype_info["data"][-1][2])
+        ntype_name = ntid_ntype_map[ntypeid]
```

Comment:
ntid_ntype_map, In this dictionary key-value pairs are ntype_id and ntype_name pairs. This dictionary is not changed. It remains the same. Except that instead of strings we now use integers as the keys for this dictionary. This was as per our last dicussion. Will update the docstring accordingly.